edge/edge-core/src/main/java/org/apache/servicecomb/edge/core/EdgeAddHeaderClientFilter.java

```diff
@@ -75,6 +75,9 @@ public boolean enabled() {
 
   @Override
   public void beforeSendRequest(Invocation invocation, HttpServletRequestEx requestEx) {
+    if (!invocation.isEdge()) {
```

Comment:
Can you add a test case for this secenio? There is a demo in demo-edge , and can easily test this scenario. demo-edge will also be run when PR is submited.

Reply:
ok